### PR TITLE
Milestone 4  Tic Tac Toe Project

### PR DESCRIPTION
**Pull request by denisdiaconu**
In this project, we recreate the Tic Tac Toe game. The game requires two participants and each one of them has to choose a symbol. In turn one, the first player needs to put his symbol in one of the positions. in turn two the second player needs to put his symbol in one of the free positions. The cycle goes on until there is a win or a tie.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,62 @@
...
+
+# Tic Tac Toe Project
+
+![screenshot](https://github.com/denisdiaconu/TicTacToe/blob/main/Screen%20Shot%202021-02-17%20at%2008.47.26.png)
+
+In this project, we recreate the Tic Tac Toe game.
+The game requires two participants and each one of them has to choose a symbol.
+In turn one, the first player needs to put his symbol in one of the positions.
+in turn two the second player needs to put his symbol in one of the free positions.
+The cycle goes on until there is a win or a tie.
```
Comment (arslanbisharat) [inline README.md]: - [x] Please remove grammar and spelling errors. You can use Grammarly for your help.

File: README.md [hunk 2]
```diff
@@ -1 +1,62 @@
...
+
+
```
Comment (arslanbisharat) [inline README.md]: - [x] You can add instructions to play game here.

File: README.md [hunk 3]
```diff
@@ -1 +1,62 @@
...
+- GitHub: [@DenisDiaconu](https://github.com/denisdiaconu)
+- Twitter: [@DenisAndreiDia1](https://twitter.com/DenisAndreiDia1)
+- LinkedIn: [Denis Diaconu](https://www.linkedin.com/in/denis-diaconu-1394091b7/)
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](issues/).
```
Comment (arslanbisharat) [inline README.md]: - [x] Please add a proper link here

**Review by arslanbisharat — CHANGES_REQUESTED** [review]
# Changes Requested :skull:

Hello, :wave:

You have done a great job. Your readme is descriptive but a few improvements can make this project ready and you can move forward with the next project. 

- [x] you have some grammar issues please use Grammarly to avoid these issues.
- [x] The instructions about how the tic-tac-toe game is played are a bit lacking in the README. Imagine if you have to explain the game to someone that never played it before, would what you have written be really enough for to someone understand the rules of the game completely?
- [ ] Commits are not descriptive. Please go back and make them descriptive. Use this [article](https://medium.com/swlh/using-git-how-to-go-back-to-a-previous-commit-8579ccc8180f). 

- [x] Link is missing. Please have a look at the inline comments. 

_Please make these changes and request another review._ :smile:

**Best,
Arslan Bisharat
slack: arslanbisharat**

File: README.md [hunk 4]
```diff
@@ -1 +1,62 @@
...
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- Hat tip to anyone whose code was used
+- Inspiration
+- etc
```
Comment (arslanbisharat) [inline README.md]: - [x] Remove this part.

**Review by arslanbisharat — CHANGES_REQUESTED** [review]

**Review by lekegitrepo — APPROVED** [review]
## Status: Approved :heavy_check_mark:
### Hi @denisdiaconu and @nganifaith :clap: :clap:
#### Good job :+1: The readme is clear and understandable now. This project is approved :white_check_mark:

#### _Reviewed by Temitayo Adeleke_

**Pull request merged**